_monitoring-plugins/alerting/security.md

```diff
@@ -91,19 +91,21 @@ In a create monitor scenario, follow these guidelines to specify roles:
 
 User type  | Role is specified by user or not (Y/N) | How to use the RBAC roles
 :--- | :--- | :---
-Admin user | User-specified | Use all the specified backend roles to associate to the monitor.
-Regular user | Yes | Use all the specified backend roles (that the user already has in their list of backend roles) to associate to the monitor.
+Admin user | Yes | Use all the specified backend roles to associate to the monitor.
+Regular user | Yes | Use all the specified backend roles from the list of backend roles that the user has permission to use to associate with the monitor.
 Regular user | No | Copy user’s backend roles and associate them to the monitor.
 
 In an update monitor scenario, follow these guidelines to specify roles:
 
 User type  | Role is specified by user or not (Y/N) | How to use the RBAC roles
 :--- | :--- | :---
-Admin user | User-specified | Remove all the backend roles associate to the monitor and then use all the specified backend roles to associate to the monitor.
-Regular user | Yes | Remove backend roles associated to the monitor that the user has access to, but didn’t specify. Then add all the other specified backend roles (that the user already has in their list of backend roles) to the monitor.
+Admin user | Yes | Remove all the backend roles associate to the monitor and then use all the specified backend roles to associate to the monitor.
+Regular user | Yes | Remove backend roles associated to the monitor that the user has access to, but didn’t specify. Then add all the other specified backend roles from the list of backend roles that the user has permission to use to the monitor.
 Regular user | No | Don’t update the backend roles on the monitor.
 
-Note that an empty list is considered the same as removing all permissions that the user possesses. Also note that if the user tries to associate roles that they don't have permission to use, it will throw an exception.
+- For admin users, an empty list is considered the same as removing all permissions that the user possesses.
```

Comment:
We need to state if a non-admin user passes in an empty list, that will throw an exception as that behavior is not allow by non-admin users.